homeassistant/components/modbus/modbus.py

```diff
@@ -166,8 +169,27 @@ def _log_error(self, exception_error: ModbusException, error_state=True):
             _LOGGER.error(log_text)
             self._in_error = error_state
 
+    def control_scan_delay(self, name, raw_scan_interval):
+        """Control the scan_intercal in the platforms."""
+        scan_interval = timedelta(seconds=raw_scan_interval)
+        if scan_interval.seconds < DEFAULT_MODBUS_TIMEOUT + 1:
+            # secure timeout is at least 1 second.
+            if scan_interval.seconds < 2:
```

Comment:
Use `max` instead.

Reply:
We normally don't allow lower scan interval than 5 seconds.

Reply:
Done.